tests/Main.hs

```diff
@@ -73,6 +73,9 @@ prop_compressThenDecompress = monadicIO . forAllM someBigString $ \ str -> do
 
 prop_decompressRandom :: Property
 prop_decompressRandom = monadicIO . forAllM someBigString $ \ str -> do
+  -- The someBigString is not necessarily big. It can even be the empty string
+  -- https://github.com/alphaHeavy/lzma-conduit/issues/19
+  pre $ B.length str > 64
```

Comment:
does it need to be >64? would any length above zero work?

```suggestion
  pre $ B.length str > 0
```

Reply:
I don't know the format of the files at all. How unlikely is it to randomly generate a valid 2 or 3 Byte file? So, I figured that 64 was large enough that a randomly generated 64 Byte file was never going to be valid.

Reply:
good point, seems like a reasonable tradeoff